core/backend/src/IModelHost.ts

```diff
@@ -673,10 +673,9 @@ export class IModelHost {
         baseUrl: credentials.baseUrl ?? `https://${credentials.account}.blob.core.windows.net`,
       },
     };
-    const ioc: Container = new Container();
-    ioc.bind<DependenciesConfig>(ExtensionTypes.dependenciesConfig).toConstantValue(config);
-    new AzureServerStorageBindings().register(ioc, config.ServerSideStorage);
-    IModelHost.tileStorage = new TileStorage(ioc.get(ServerStorage));
+    const ioc = InversifyWrapper.create();
+    ioc.registerInstance<DependenciesConfig>(ExtensionTypes.dependenciesConfig, config);
+    IModelHost.tileStorage = new TileStorage(ioc.resolve<ServerStorage>(ExtensionTypes.serverStorage));
```

Comment:
Do we need to use `inversify` ?

Since method is called `setupAzureTileCache`, I assume this part is well aware of what provider (Azure) it's using. We could try creating `AzureServerStorage` directly ([Gist](https://gist.github.com/GytisCepk/bcaf0921a082feb502e640a52f17c585))